### PR TITLE
Adding Prim's algorithm in JS (Issue #2201)

### LINKED ISSUE #2201: Prims Algorithm
- [x] C
- [x] C++
- [ ] C# - @ritish099
- [ ] CoffeeScript
- [ ] Dart - @Swarnimashukla
- [ ] Go - @adi44
- [x] Java
- [x] JS - @sukritishah15
- [x] Kotlin - @darkmatter18
- [ ] Php - @prince-09
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript - @BhuwanChandra

### PR DESCRIPTION
**Pull request by sukritishah15**
Fixes #2201 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- JS

#### Files Added:
- Prims_Algorithm.js

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by sukritishah15** [comment]
@raghav-dalmia @asha15 @SakshiShreya @MastersAbh @AdityaShD @jainaman224 

Kindly review my PR.

File: Prims_Algorithm/Prims_Algorithm.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    this.addEdge = function(first, second, weight) {    
+        this.edges[first].push(new Edge(first, second, weight));
+        this.edges[second].push(new Edge(second, first, weight));
+    };
+};
+
+// To get nearest vertex to visited vertices
+function getMinVertex(g,result,visitedVertices) {
+    var min = [Infinity , null];
+    for(var i=0; i<result.length; i++)
```
Comment (singh-shreya6) [inline Prims_Algorithm/Prims_Algorithm.js:32]: Please add space around operators like =, <

File: Prims_Algorithm/Prims_Algorithm.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+g.addVertex(3);
+
+// Weighted edges for the graph - Sample Input
+g.addEdge(0, 1, 3);
+g.addEdge(1, 3, 1);
+g.addEdge(1, 2, 5);
+g.addEdge(2, 3, 8);
+
+var result = Prims(g);
+document.write('<h4> Order in which vertices are visited during the implementation of Prim\'s Algorithm: </h4>')
```
Comment (singh-shreya6) [inline Prims_Algorithm/Prims_Algorithm.js:74]: You can use console.log here. HTML tags won't be required then.

File: Prims_Algorithm/Prims_Algorithm.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+document.write(result);
+
+/*
+    Sample Input is as taken in the code.
+    Sample Output:
+    Order in which vertices are visited during the implementation of Prim's Algorithm:
+    0,1,3,2
+*/
+</script>
```
Comment (singh-shreya6) [inline Prims_Algorithm/Prims_Algorithm.js:83]: script tag is not needed.

File: Prims_Algorithm/Prims_Algorithm.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+<script>
```
Comment (singh-shreya6) [inline Prims_Algorithm/Prims_Algorithm.js:1]: script tag is not needed

File: Prims_Algorithm/Prims_Algorithm.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+// Weighted edges for the graph - Sample Input
+g.addEdge(0, 1, 3);
+g.addEdge(1, 3, 1);
+g.addEdge(1, 2, 5);
+g.addEdge(2, 3, 8);
+
+var result = Prims(g);
+document.write('<h4> Order in which vertices are visited during the implementation of Prim\'s Algorithm: </h4>')
+document.write(result);
+
```
Comment (singh-shreya6) [inline Prims_Algorithm/Prims_Algorithm.js:76]: Use console.log to print things.

**Comment by sukritishah15** [comment]
@singh-shreya6 I have done the requested changes and squashed the commits. Kindly re-review the PR. Thank you!

**Comment by sukritishah15** [comment]
@singh-shreya6 I have done the requested changes and squashed the commits. Kindly re-review the PR. Thank you!

**Pull request merged**